lua/cfc_random_spawn/sh_config.lua

```diff
@@ -3,20 +3,22 @@ cfcRandomSpawn.config.MAXIMUM_DISTANCE_FROM_PLAYERS = 500 -- The maximum distanc
 
 cfcRandomSpawn.config.CUSTOM_SPAWNS = {}
 cfcRandomSpawn.config.CUSTOM_SPAWNS["gm_bluehills_test3"] = {
-	Vector( 1104.6744384766, 131.18112182617, 64.03125 ),
-	Vector( 385.55798339844, 157.09564208984, 64.03125 ),
-	Vector( 1272.7860107422, -485.29537963867, 264.03125 ),
-	Vector( 1117.4444580078, -1345.0144042969, 64.03125 ),
-	Vector( 1458.7664794922, -484.67016601563, 64.03125 ),
-	Vector( 971.29565429688, 401.40603637695, 264.03125 ),
-	Vector( 392.85052490234, 146.80117797852, 264.03125 ),
-	Vector( -456.48602294922, -442.73083496094, 264.03125 ),
-	Vector( 1148.5478515625, 90.367889404297, 456.03125 ),
-	Vector( -214.91020202637, -623.013671875, 64.03125 ),
-	Vector( -726.35272216797, -540.54400634766, 64.03125 ),
-	Vector( -809.68939208984, 1255.5084228516, 464.03125 ),
-	Vector( 425.38064575195, 1636.2911376953, 64.968605041504 ),
-	Vector( -974.1875, 130.53991699219, 128.03125 ),
-	Vector( -1357.5144042969, 1473.0662841797, 128.03125 ),
-	Vector( -1000.3621826172, 1044.5682373047, 464.03125 )
+
+	{ "spawn" = Vector( 1104.6744384766,  131.18112182617,  64.03125 ),        "pvp" = true },

```

Comment:
I don't like these keys. Could we do:

```lua
{ "spawn_pos" = Vector( ), "isPvpSpawn" = true }
```
Something like that. 

Mostly this helps the other file read better.